### PR TITLE
Add separated dir for gem index

### PR DESCRIPTION
**Pull request by Sammers21**
SubmitGem creates gem index dir when constructed

Depends on #18

### CODE REVIEW

File: src/main/ruby/SubmitGem.rb [hunk 1]
```diff
@@ -8,10 +9,16 @@ class SubmitGem
...
+
   @@log = LoggerFactory::getLogger("com.artipie.gem.SubmitGem")
 
   def initialize(storage)
     @storage = storage
+    idx = "temp-gem-index"
+    gems = File.join(idx, "gems")
+    Dir.mkdir(idx) unless File.exists?(idx)
+    Dir.mkdir(gems) unless File.exists?(gems)
+    Gem::Indexer.new(idx, { build_modern: true })
```
Comment (g4s8) [inline src/main/ruby/SubmitGem.rb:21]: @Sammers21 it seems you need to assign this object to some variable, isn't it?

**Comment by 0crat** [comment]
This pull request [#21](https://github.com/artipie/gem-adapter/issues/21) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/9e122ff8-5fea-46f1-acf2-39a7693161ac); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @g4s8[/z](https://www.0crat.com/u/g4s8)[/z](https://www.0crat.com/u/g4s8) didn't make CR comments

**Comment by 0crat** [comment]
Code review was too long (8 days), architects (@g4s8) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Pull request merged**